documentation/player-portal/web/v2/api.md

```diff
@@ -0,0 +1,1886 @@
+
+[Source](http://player.kaltura.com/docs/api "Permalink to Kaltura Player - Fast, Flexible, Video Player Toolkit")
+
+# Kaltura Player - Fast, Flexible, Video Player Toolkit
+
+This documentation covers version **_2.74.1__1e322ce7_** of the html5 library. 
+
+The kWidget API is available after you include the Kaltura player library. kWidget provides embedding and basic utility functions. The Kaltura player library can be embeded into both 
+
+. 
+
+After you include the Kaltura player library, the following kWidget API is available: 
+
+* * *
+
+#### kWidget.embed ( [targetId], settings)
+
+Used to embed the Kaltura player against an element target in the DOM
+
+##### PARAMETERS:
+
+* **targetId** String _( Optional )_ The DOM player target id attribute string. ( if not included, you must include targetId in "settings" object )
+* **settings** [kWidget.settingsObject][1] Object of settings to be used in embedding.
+
+##### RETURNS:
```

Comment:
See previous note about using unique headers due to the auto creation of anchors.